.github/workflows/ci-workflow.yml

```diff
@@ -228,9 +230,10 @@ jobs:
           restore-keys: |
             mvn-it-
       - name: Set up JDK 1.8 JP
```

Comment:
Is it worth calling out the distribution in the job name now that it is different?